server/service/service_campaigns.go

```diff
@@ -135,6 +135,14 @@ func (svc Service) NewDistributedQueryCampaign(ctx context.Context, queryString
 	if err != nil {
 		return nil, errors.Wrap(err, "counting hosts")
 	}
+
+	if err := svc.ds.NewActivity(
+		authz.UserFromContext(ctx),
+		fleet.LiveQueryActivityType,
+		&map[string]interface{}{"target_counts": len(targets.HostIDs)},
```

Comment:
```suggestion
		&map[string]interface{}{"target_counts": len(campaign.Metrics.TotalHosts)},
```
(`len(targets.HostIDs)` returns the count of explicitly targeted hosts, but does not include the counts of hosts that have been included via Label or Team.